pyseir/ensembles/ensemble_runner.py

```diff
@@ -58,7 +112,7 @@ class EnsembleRunner:
 
     def __init__(
         self,
-        fips,
+        regional_input: RegionalInput,
```

Comment:
Not 100% sure where to put this comment, but I'm not sure how I feel about `region` being strictly accessed through `regional_input`.  My thinking is that keeping the two separate might be cleaner: `region: Region, input_data: RegionalInput`. Having to get fips and state through nested calls like `regional_input.region.fips` feels a bit of a code smell where as having `region` as its own input to the function may be more clean?  I feel like region could definitely still be on the `RegionalInput` class (maybe as `_region`), but maybe we surfacing things from like `regional_input.fips` might be more clean?

Reply:
I'm not a fan of passing around two objects in parallel but added RegionalInput.fips (as a property!) and state_name to reduce the r_i.region.X calls, which I agree are a little ugly.